cocos/ui/UIScrollView.cpp

```diff
@@ -1342,8 +1342,8 @@ void ScrollView::copySpecialProperties(Widget *widget)
     if (scrollView)
     {
         Layout::copySpecialProperties(widget);
-        _innerContainer = scrollView->_innerContainer;
         setDirection(scrollView->_direction);
+        setInnerContainerPosition(scrollView->getInnerContainerPosition());
```

Comment:
I think it's more reasonable to do the following:

_innerContainer->copySpecialProperties(widget->_innerContainer);